package.json

```diff
@@ -57,7 +57,7 @@
     "@babel/preset-react": "^7.16.7",
     "@babel/preset-typescript": "^7.16.7",
     "@svgr/webpack": "^6.0.0",
-    "@types/history": "^5.0.0",
+    "@types/history": "^4.7.11",
```

Comment:
I think Dependabot keeps bumping this dependency, but we want it back at major version 4 to align with our version of history that we need to keep for `connected-react-router`.